BookReader/plugins/plugin.autoplay.js

```diff
@@ -56,9 +56,11 @@ BookReader.prototype.bindNavigationHandlers = (function(super_) {
  * Starts autoplay mode
  * TODO move to a plugin
```

Comment:
Just noticed this PR. Looks good for the most part. You can remove the TODO comment, since it's in a plugin now.

Also, one suggestion would be to take explicit params in the autoToggle function, rather than a generic options. This locks down the methods signature. If you do want to take options, consider having a set of defaultOptions within the method, so that it's clear what type of options are supported.

Reply:
I initially had them broken out as separate parameters, but I didn't want to force users to pass a value for the first one if they only wanted to override the second. I'll put in some defaults and extend the options on them.